api/app.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask
+from flask import request
+from flask import url_for
+from geodepy.geodesy import vincdir
+from geodepy.geodesy import vincinv
+from geodepy.convert import dms2dd
+from geodepy.convert import dd2dms
+
+app = Flask(__name__)
+
+coord2dd = {
+    'dd': lambda x: x,
+    'dms': dms2dd,
+}
+
+dd2coord = {
+    'dd': lambda x: x,
+    'dms': dd2dms,
+}
+
+
+@app.route('/')
+def list_routes():
+    return str(tuple(url_for(rule.endpoint) for rule in app.url_map.iter_rules() if rule.endpoint != 'static'))
```

Comment:
I wonder if there's a framework for python like the one used at https://gws.geodesy.ga.gov.au/ which would give us the routes in a nicer format.

Reply:
Yeah, I was looking for one but couldn't find it.